extensions/wikia/Chat2/ChatEntryPoint.class.php

```diff
@@ -23,35 +23,56 @@ static public function onParserFirstCallInit( Parser &$parser ) {
 	}
 
 	/**
-	 * @param $isEntryPoint - set to true for chat entry point embeded in the article, false for rail module
-	 * Return an array of variables needed to render chat entry point template
+	 * @param $isEntryPoint - set to true for chat entry point embedded in the article,
+	 * set to false for rail module
+	 *
+	 * Return an array of variables needed to render chat entry point mustache template
```

Comment:
Most of the time the main description comes first, then you list parameters and the return value at the end